Makefile

```diff
@@ -300,10 +301,8 @@ install: build
 ###### TARGETS FOR CICD PROCESS ######
 include ./scripts/release/mule/Makefile.mule
 
-SUPPORTED_ARCHIVE_OS_ARCH = linux/amd64 linux/arm64 linux/arm darwin/amd64
-
 archive:
 	CHANNEL=$(CHANNEL) \
-	PATH=$(SRCPATH)/tmp/node_pkgs/$(OS_TYPE)/$(ARCH)/bin:$${PATH} \
-	scripts/upload_version.sh $(CHANNEL) $(SRCPATH)/tmp/node_pkgs $(S3_RELEASE_BUCKET)
+	FULLBUILDNUMBER=$(FULLBUILDNUMBER) \
+	aws s3 cp tmp/node_pkgs s3://algorand-internal/channel/${CHANNEL}/${FULLBUILDNUMBER} --recursive --exclude "*" --include "*${CHANNEL}*${FULLBUILDNUMBER}*"
```

Comment:
Is it necessary to create `FULLBUILDNUMBER` here? Can you just do:
```
s3://algorand-internal/channel/${CHANNEL}/$(FULLBUILDNUMBER)
```

Reply:
i think so, i'll try changing that 👍

Reply:
Updated